xml/dm_registers.xml

```diff
@@ -508,15 +508,40 @@
 
     <!-- =============== system bus mastering =============== -->
 
+    <register name="System Bus Address 127:96" short="sbaddress3" address="0x37">
+        If \Fsbasize is less than 97, then this register is not present.
+
+        <field name="address" bits="31:0" access="R/W" reset="0">
+            Accesses bits 127:96 of the physical address in {\tt sbaddress} (if
+            the system address bus is that wide).
+        </field>
+    </register>
+
     <register name="System Bus Access Control and Status" short="sbcs" address="0x38">
-        <field name="0" bits="31:21" access="R" reset="0" />
-        <field name="sbsingleread" bits="20" access="W1" reset="0">
-            When a 1 is written here, triggers a read at the address in {\tt
-            sbaddress} using the access size set by \Fsbaccess.
+        <field name="version" bits="31:29" access="R" reset="1">
+            0: The System Bus interface conforms to mainline drafts of this
+            spec older than 1 January, 2018.
+
+            1: The System Bus interface conforms to this version of the spec.
+
+            Other values are reserved for future versions.
+        </field>
+        <field name="0" bits="28:22" access="R" reset="0" />
+        <field name="sbbusy" bits="21" access="R" reset="0">
+            When 1, indicates the system bus master is busy. This bit goes high
+            immediately when a read or write is requested for any reason, and
+            does not go low until the access is fully completed.
+
+            To avoid race conditions, debuggers must not try to clear \Fsberror
+            until they read \Fsbbusy as 0.
+        </field>
+        <field name="sbreadonaddr" bits="20" access="R/W" reset="0">
+            When a 1, every write to \Rsbaddresszero automatically triggers a
+            system bus read at the new address.
         </field>
         <field name="sbaccess" bits="19:17" access="R/W" reset="2">
             Select the access size to use for system bus accesses triggered by
-            writes to the {\tt sbaddress} registers or \Rsbdatazero.
+            writes to \Rsbaddresszero or \Rsbdatazero.
```

Comment:
only writes? What about reads to \Rsbdatazero?

Reply:
This language is really old. It controls the size of every system bus access. Fixing...